frontend/src/pages/playback/music/index.vue

```diff
@@ -50,11 +51,17 @@
             <time-slider />
           </v-row>
           <v-row class="justify-center align-center">
+            <audio-visualise-button size="x-large" />
```

Comment:
In my opinion, this button should be at the top right in the app bar, since it's not related to playback at all, it's just aesthetic.

Reply:
I agree that it should probably be elsewhere, but I feel that it would be even more out of place in the top right since it would be floating alone, in my mind it would make more sense if there was already something there like a casting button. The placement doesn't bother me that much though, so I don't mind changing it if you want.